pkg/netlight/public/public_test.go

```diff
@@ -27,6 +28,12 @@ func TestCreatePublicNS(t *testing.T) {
 		require.NoError(t, err)
 	}()
 
-	err := setupPublicNS(pkg.StrIdentifier(""), iface)
+	_, err := bridge.New("br-pub")
+	require.NoError(t, err)
+	defer func() {
+		err = bridge.Delete("br-pub")
+		require.NoError(t, err)
+	}()
+	err = setupPublicNS(pkg.StrIdentifier(""), iface)
```

Comment:
you can use `ensurePublicBridge` instead, please don't use `pr-pub` if needed use the `PublicBridge` exported const
https://github.com/threefoldtech/zos4/blob/52bec593eae75cffcd1b9eae60989b40713be1b3/pkg/netlight/types/constant.go#L15
https://github.com/threefoldtech/zos4/blob/52bec593eae75cffcd1b9eae60989b40713be1b3/pkg/netlight/public/public.go#L46